data/hai/unfettered jobs.txt

```diff
@@ -263,16 +263,16 @@ mission "Unfettered Medicals"
 	on visit
 		dialog phrase "generic missing stopover or cargo"
 	on stopover
-		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong," they ask while loading the supplies on your ship.`
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong," one says while loading the supplies onto your ship.`
 	on complete
 		payment 10000 4000
-		dialog `The local health specialists seem relieved that you have managed to bring the needed medical supplies back in time.`
+		dialog `The local health specialists are relieved that you have managed to bring the needed medical supplies back in time.`
 
 
 
 mission "Unfettered Challenge [0]"
-	name `Defeat the young captain`
-	description `A young Unfettered captain has provoked you in a duel. Disable his ship for <payment>.`
+	name `Defeat young Unfettered captain`
```

Comment:
I would have thought a 'the' would have been required here, but you're the grammar expert